realm/src/androidTest/java/io/realm/RealmLinkTests.java

```diff
@@ -463,4 +463,29 @@ public void testSubquery() {
         RealmResults<Owner> subOwners = owners.where().equalTo("cat.name", "Blackie").findAll();
         assertEquals(1, subOwners.size());
     }
+
+    public void testIsNull() {
+        RealmResults<Owner> owners1 = testRealm.where(Owner.class).isNull("dogs").findAll();
```

Comment:
I don't think that a link list can ever logically be null (it can be empty
though).

On Friday, March 20, 2015, Emanuele Zattin <notifications@github.com> wrote:

> In realm/src/androidTest/java/io/realm/RealmLinkTests.java
> <https://github.com/realm/realm-java/pull/980#discussion_r26848052>:
>
> > @@ -463,4 +463,29 @@ public void testSubquery() {
> >          RealmResults<Owner> subOwners = owners.where().equalTo("cat.name", "Blackie").findAll();
> >          assertEquals(1, subOwners.size());
> >      }
> > +
> > +    public void testIsNull() {
> > +        RealmResults<Owner> owners1 = testRealm.where(Owner.class).isNull("dogs").findAll();
>
> I think we should also test on a null link and linklist
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/realm/realm-java/pull/980/files#r26848052>.
>


-- 
*Alexander Stigsen, CEO*
Realm Inc

148 Townsend St
San Fransisco, CA 94107
United States